tests/network-tests/src/flows/storage/initStorage.ts

```diff
@@ -35,6 +38,16 @@ export const allStaticBags: CreateInterface<StaticBagId>[] = [
   { WorkingGroup: 'Storage' },
 ]
 
+const sampleStorageBucket = {
+  metadata: { endpoint: process.env.COLOSSUS_1_URL || 'http://localhost:3333' },
+  staticBags: allStaticBags,
+  operatorId: parseInt(process.env.COLOSSUS_1_WORKER_ID || '0'),
+  storageLimit: new BN(1_000_000_000_000),
+  objectsLimit: 1000000000,
+  transactorUri: process.env.COLOSSUS_1_TRANSACTOR_URI || '//Colossus1',
+  transactorBalance: new BN(100_000),
```

Comment:
I introduced this as just an example value when feed were not yet set, but now during the tests I noticed it's very low with our current feed and JOY denomination. I suggest setting it to at least `10_000_000_000` (1 JOY) for all configs

Reply:
Addressed in [b552fba](https://github.com/Joystream/joystream/pull/3997/commits/b552fba6a40a75129fb358c4b771150313077472)